disnake/member.py

```diff
@@ -794,6 +818,15 @@ async def edit(
         if roles is not MISSING:
             payload["roles"] = tuple(r.id for r in roles)
 
+        if communication_disabled_until is not MISSING:
+            # TODO: Add audit logs.
+            if communication_disabled_until is not None:
+                payload["communication_disabled_until"] = communication_disabled_until.astimezone(
+                    tz=datetime.timezone.utc
+                ).isoformat()
+            if communication_disabled_until is None:
```

Comment:
```suggestion
            else:
```

Reply:
Before implementing this, I would like to handle the user input, if I send a type that isn't datetime, it raises: `{type} object has no attribute 'astimezone'`.

I'm not sure if this is a good idea.